### PR TITLE
Refactor Simple Chat App tutorial

### PR DESCRIPTION
**Pull request by gmelodie**
This relates to #798. I had a few thoughts but figured it'd be easier to create a PR so you can see what I mean more concretely rather than discussing it on the issue. Hope you enjoy it. Looking forward to hearing back :)

Below you'll find a summary of changes I made.

**Getting the code**
1. Move live demo link from "Getting the code" to very top section.

**Peer discovery and connectivity:**
1. Add links to circuit relay docs and WebRTC-star
2. Color indicators are now an unordered list

**Docker**
1. Move "After this section ..." to the section above
2. Include a link to certbot
3. Replace `DOMAIN.COM` with `YOURDOMAIN.COM` (more obvious, and used later on)

**WebRTC-Star**
1. Update links to webrtc-star docs (currently archived and deprecated, might be worth to revisit)

**p2p-circuit**
1. Add a better intro to shortly explain why we would need circuits and how it works in a high level
2. Reduce text from usage explanation to make it cleaner
3. Add tip about the `ipfs.thedisco.zone` domain
4. Edit setup description to shortly explain websockets beforehand
5. Make advertising script section more interactive (more steps)

**Communication**
1. `discochat-global` typo, should be `example_topic` as per code

**Possible browser pitfalls**
1. Make text under "Staying connected to peers" more direct

### CODE REVIEW

**Comment by lidel** [comment]
@gmelodie did you had chance to test with go-ipfs 0.11, or have bandwidth to update this tutorial?

TLDR: enabling relay v1 with  `EnableRelayHop` no longer works in go-ipfs 0.11.0 (see [breaking changes](https://github.com/ipfs/go-ipfs/releases/tag/v0.11.0)). We need to update this tutorial to point folks at relay v1 from js-ipfs or a standalone [libp2p-relay-daemo running as RelayV1](https://github.com/libp2p/go-libp2p-relay-daemon#minimal-config-file).

**Comment by BigLep** [comment]
@gmelodie : are you able to incorporate the feedback?  Thanks.

**Comment by gmelodie** [comment]
Hey! I'm a bit busy with work for now, if someone's able to make those changes that'd be awesome!

**Comment by johnnymatthews** [comment]
If this is still being worked on, I'm gonna change this to a draft.

**Comment by TMoMoreau** [comment]
With a new version of the chat app, an entirely new guide/tutorial is needed, see issue #1459. Closing this PR.
